source/Engine.cpp

```diff
@@ -100,6 +101,26 @@ namespace {
 		return Radar::UNFRIENDLY;
 	}
 
+	// TODO (C++20): Replace with erase_if.
```

Comment:
this comment belongs to  the other prune i think

Reply:
Can't we replace all of them?

Reply:
I am not quite sure, for some reason this one was made more complex, it already uses its own remove_if for some reason